integrations/whatsapp/src/index.ts

```diff
@@ -16,8 +17,26 @@ const { Text, Media, Location } = Types
 const integration = new bp.Integration({
   register: async () => {},
   unregister: async () => {},
-  actions: {},
-  createConversation,
+  actions: {
+    startConversation: async ({ ctx, input, client, logger }) => {
```

Comment:
**nit:** maybe extract this in a dedicated file or `conversation.ts` file with type `botpress.IntegrationProps['actions']['startConversation']`

Reply:
I don't really like extracting simple/short logic into a separate file for integrations as I find it very valuable to see at least some of the top-level logic right away, so I can just open the integration's index.ts file and get a quick idea of what it's doing by just scrolling it instead of having to inspect multiple imports.